src/base/solvers/solveCobraLP.m

```diff
@@ -1119,6 +1123,9 @@
             tomlabProblem.MIP.cpxControl.EPOPT = optTol;
         end
 
+        % set parameters
+        tomlabProblem.MIP.cpxControl = solverParams;
+        
```

Comment:
This overwrites any existing settings set above (and it always does so independent on whether the struct was provided or not). 
You would need a test to see if the struct was provided (which is probably a pain due to the messy input structure), or this will make other option settings impossible and have tomlab always run with default options.

Reply:
:+1: